services/ui-src/src/components/forms/AdminDashSelector.tsx

```diff
@@ -33,10 +33,17 @@ export const AdminDashSelector = ({ verbiage }: Props) => {
 
   // assemble and inject report choices depending on whether report is enabled
   const mlrReport = useFlags()?.mlrReport;
+  const reportField = formJson.fields.find((field) => field.id === "report")!;
+
   if (mlrReport) {
+    reportField.type = "radio";
+    reportField.validation = "radio";
     reportChoices.push(mlrReportChoice);
+  } else {
+    reportField.type = "checkbox";
```

Comment:
Fixes `[Warning]: Use type="checkbox" for components with only one choice. A single radio button is disallowed because it prevents users from deselecting the field.`

Reply:
🤔 huh the MLR is enabled in all envs. I bet we could remove this flag. That said, fine to fix it here for now

Reply:
I'll write up a ticket for removing the flag throughout the app.